.gitignore

```diff
@@ -2,7 +2,5 @@
 .Rhistory
 .RData
 .Ruserdata
-*.png
-*.txt
-*/out/*.csv
-
+_targets
```

Comment:
I was wrong about this, sorry!. The `_targets/` directory will have to be removed from git entirely, or else when someone runs the pipeline new versions of the files will be created and git will prompt to commit them.

So can you remove them with `git rm`, and then commit the removal? No need to worry about removing this directory: pipelines will happily recreate it.